webapp/models.py

```diff
@@ -121,7 +129,7 @@ class JiraTask(db.Model, DateTimeMixin):
     jira_id: int = Column(Integer)
     webpage_id: int = Column(Integer, ForeignKey("webpages.id"))
     user_id: int = Column(Integer, ForeignKey("users.id"))
-    status: str = Column(String)  # Will be filled from Jira API
+    status: str = Column(Enum(JIRATaskStatus), default=JIRATaskStatus.TRIAGED)
     created_at: str = Column(String)
```

Comment:
this column stays `null` after a new row is added

Reply:
I've removed the enum, and now using plain strings

Reply:
@samhotep this is still an issue, after a row is added to "jira_tasks" table, the "created_at_ column is `null`

Reply:
I've fixed it, removing the extra created_at declaration did the trick